style.css

```diff
@@ -11,238 +11,400 @@
 }
 
 body {
-  display: flex;
-  flex-direction: column;
-  max-width: 375px;
-  font-family: 'Poppins', sans-serif;
   background: #f7f7f9;
   margin: auto;
-  border: 2px solid #ccc;
 }
 
-.heading {
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
-  color: #172b4d;
-  text-transform: capitalize;
+.headr {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  background: #fff;
+  padding: 26px 7%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
 
-.btn {
-  margin-top: 1rem;
-  cursor: pointer;
-  text-transform: capitalize;
-  border: 1px solid #6070ff;
-  border-radius: 8px;
+.headr .logo {
   font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  text-align: center;
-  letter-spacing: 0.03em;
-  color: #396df2;
-  padding: 12px;
-  gap: 10px;
-  width: 124px;
-  background: #fff;
+  font-weight: 700;
```

Comment:
- Great job for using the exact `font-weight: 700;`😁👍 however it is not applying because you didn't import the exact font, Please be sure to import the exact weight from [google font](https://fonts.google.com/) before using it 👍👌